app/controllers/network_services_controller_test.go

```diff
@@ -0,0 +1,519 @@
+package controllers
+
+import (
+	"fmt"
+	"github.com/cloudnativelabs/kube-router/app/watchers"
+	"github.com/docker/libnetwork/ipvs"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/vishvananda/netlink"
+	v1core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+	"net"
+	"time"
+)
+
+type LinuxNetworkingMocker interface {
```

Comment:
This seems like a duplicate of `LinuxNetworking` can we just use that?

Reply:
Good catch, done ( fwiw I had slightly preferred explicitly having there
the `*Mock()` methods as a way to document~ish what subset has
been mocked).